src/adapters/leaderboard_adapters.py

```diff
@@ -35,3 +36,22 @@ def league_standing_to_pretty_table(standing: LeagueStanding) -> pt.PrettyTable:
     for e in standing.entrants:
         table.add_row([e.user.username, e.score])
     return table
+
+
+def entrant_to_pretty_input(standing: LeagueStanding) -> List[InlineKeyboardButton]:
+    keyboard = []
+    tmp = []
+    for i in range(len(standing.entrants)):
+        if i % 3 == 0:
+            keyboard.append(tmp)
+            tmp = []
+        # Skip fake user
+        if standing.entrants[i].user.username != "Asdfgh S.":
+            tmp.append(
+                InlineKeyboardButton(
+                    standing.entrants[i].user.username,
+                    callback_data=standing.entrants[i].user.global_id,
+                )
+            )
+    keyboard.append(tmp)
+    return keyboard
```

Comment:
I'm a bit confused about this method 😕

Reply:
This method will build a matrix with 3 columns. Every 3 users, a new line is created. It's just to make the output clear